Real aggregate

dkaygithub opened the pull request:
completed aggregation UI

ssaamm commented:
Squash these into 1 commit

ssaamm commented on js/chartjs/README.md:
```diff
@@ -0,0 +1,20 @@
+# Chart.js 
```
Do we need the Chart.js readme in our repo?

ssaamm commented on js/chartjs/docs/00-Getting-Started.md:
```diff
@@ -0,0 +1,215 @@
+---
```
I don't think we need the entire `chartjs/docs` folder

ssaamm commented on js/chartjs/samples/doughnut.color.html:
```diff
@@ -0,0 +1,58 @@
+<!doctype html>
```
I don't think we need the entire `js/chartjs/samples` folder

ssaamm commented on js/chartjs/src/Chart.Bar.js:
```diff
@@ -0,0 +1,303 @@
+(function(){
```
We probably don't need the source code...? Remove the `js/chartjs/src/` folder.

ssaamm commented on templates/userAggregate.html:
```diff
@@ -0,0 +1,230 @@
+{% set active_page = "query" %}
+{% block title %}Reports{% endblock %}
+{% block head_extra %}
+        <script src="/buoy.git/js/chartjs/chart.min.js"></script>
```
This should be `js/chartjs/chart.min.js`, no? Relative paths are important.

ssaamm commented on userAggregateSimple.php:
```diff
@@ -0,0 +1,116 @@
+<?php
```
We don't need this file in here anymore, right? We have a (hopefully working) `userAggregate.php` now.

Commit:
more cleanup

added aggregate simple page

put aggregate in UI with table

bad code, but have to commit sometime. need to find bug

made gitignore

more gitignore

removing the compilation_cahce files

working for year,month,and week

removed excess chart.js paraphanelia
remove debugging

dkaygithub commented:
i think it's ready

ssaamm commented:
Squash the commits?

dkaygithub commented:
how?
I know how to do it while i'm working on stuff, but after pushing i don't know how to do it in the ui

ssaamm commented:
`git rebase -i HEAD~10`

dkaygithub commented:
i will try, havent done command line in forever
